app/invocation/invocation_action_card.tsx

```diff
@@ -316,13 +318,28 @@ export default class InvocationActionCardComponent extends React.Component<Props
       })
       .catch((e) => {
         const error = BuildBuddyError.parse(e);
-        if (error.code === "NotFound") {
+        if (error.code === "NotFound" || error.code === "Unimplemented") {
```

Comment:
I'm a bit confused why this change is needed - I would think that `executeResponseDigest` shouldn't be set in the URL when reaching this page from the cache requests card?